mkosi/backend.py

```diff
@@ -89,6 +89,16 @@ def __str__(self) -> str:
         return self.name
 
 
+class Compression(str, enum.Enum):
+    none = "none"
+    zst = "zst"
+    xz = "xz"
+    bz2 = "bz2"
+    gz = "gz"
+    lz4 = "lz4"
+    lzma = "lzma"
```

Comment:
I don't think this being a str subclass is used anywhere
```suggestion
class Compression(enum.Enum):
    none = None
    zst = "zst"
    xz = "xz"
    bz2 = "bz2"
    gz = "gz"
    lz4 = "lz4"
    lzma = "lzma"
    
    def __str__(self):
        return str(self.value).lower()
        
    def __bool__(self):
        return bool(self.value)
```
will produce the same stringifcation, but you can call `Compression(None)` directly, which might be helpful for the argument parser and the bool would make it intuitively usable, because `Compression.none` would be falsey and the others truthy.

Reply:
Sure, no problem.